Automattic/wp-calypso
# Implement url analyzer API for import onboarding

#### Changes proposed in this Pull Request

This change implements the API endpoint for the URL analyzer in the import onboarding flow at `/start/import/capture`

#### Testing instructions

 - Go to `/start/import/capture`.
 - Enter a valid URL and hit enter
 - The scanning step will show - possibly really quick - followed by the preview page.
 - Check if the site address and the platform are correctly displayed in the content below the page title.

Not all functionality has been fully implemented yet. This PR purely focuses on getting the data from the URL analyzer endpoint and making it available in the step components.

Related to #57131
